test/framework/debug.go

```diff
@@ -93,16 +181,15 @@ func DebugKube(cluster Cluster, mesh string, namespaces ...string) {
 
 	exportFilePath := filepath.Join(Config.DebugDir, fmt.Sprintf("%s-export-%s", cluster.Name(), uuid.New().String()))
 	Expect(os.WriteFile(exportFilePath, []byte(out), 0o600)).To(Succeed())
-	Expect(errorSeen).To(BeTrue(), "some debug commands failed")
+	Expect(errorSeen).NotTo(BeTrue(), "some debug commands failed")
```

Comment:
I'm assuming this is just a mistake, which I fixed, because it's correct for universal: https://github.com/kumahq/kuma/pull/11223/files#diff-fa3b44fc3afa9217ddbcdd2ed1527380c7829f4ba2c402a73f95227ea43f5923L35